Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
+FROM python:3.10-alpine
+
+WORKDIR /tmp/indeed
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apk update && apk add tk
+RUN apk add font-terminus font-inconsolata font-dejavu font-noto font-noto-cjk font-awesome font-noto-extra
```

Comment:
We can probable remove most of these dependencies. The alpine image is missing one of these packages to launch the gui.

Reply:
Noted.